### DISCUSSION TITLE
[feat-req] switch to other screen

### DISCUSSION THREAD

**Discussion by tx46** [qtile/qtile#3523] [Ideas]
Hello,

I have a multi-monitor setup with Qtile. It's working fine - I've written my own sort-of "alt-tab" functionality which works well to switch back and forth between windows on the same screen, as well as switching between windows on different screens. That only works, however, after windows have been focused.

What I'd like to do is to set up a hotkey to merely "switch to the other screen." For example, let's say I am looking at my left monitor. I would like to hit whatever keyboard shortcut, and have the focus jump to whatever is on my right monitor instead.

Is this possible today, or how can I go about doing it?

**Comment by SweepingBishops** [comment]
Set a keybinding for `lazy.next_screen()`.

**Comment by asyncmind0** [comment]
I have single key binding for jumping to specific screen and group etc ... 
https://github.com/jagguli/TAQtile/blob/master/taqtile/keys.py#L300

**Comment by diego-rapoport** [comment]
If I remember well, a few versions ago there was a  default using the key bind you would use for the same group you were already.
Example: you have monitor 1 with group 2 and monitor 2 with group 5. If on monitor 1 you hit the binding to group 2, it would swap groups on monitors. The same would also happen if you were on monitor 2 and hit the binding for group 5.
I think there was some complaining about it, but that was a feature I used a lot. Didn't care that much because I thought that people coming from other WM's felt like this was an odd behavior.

**Reply by m-col** [reply]
Do you mean the `toggle` option for `lazy.group.toscreen()`, i.e. `lazy.group.toscreen(toggle=True)`? This was true by default until a couple versions ago, but it was changed to false. You can pass it like that to get the behaviour back. It was changed because when it was first introduced (only a couple versions before) it was an unexpected change for many people.

**Reply by diego-rapoport** [reply]
Thank you very much, @m-col !!! Sure will toggle that to true!
